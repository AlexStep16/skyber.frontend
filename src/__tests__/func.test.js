import { mount } from '@vue/test-utils'
import Component from '@/components/DeleteModal.vue'

describe('Component', () => {
  test('Удаление модальное окно', () => {
    const wrapper = mount(Component, {
      propsData: {
        message: 'qdw'
      }
    })
    expect(wrapper.element).toMatchSnapshot()
  })
})
