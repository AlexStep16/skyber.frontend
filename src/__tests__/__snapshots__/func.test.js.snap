// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component является экземпляром Vue 1`] = `
<div
  class="modal modal_black flex flex-center"
>
  <div
    class="modal-container"
  >
    <div
      class="modal-confirm-block flex flex-vertical flex-align-center"
    >
      <h2
        class="modal-confirm-block__message"
      >
        qdw
      </h2>
       
      <div
        class="mt5"
      >
        <button
          class="button-clasic button_theme-red button_theme-red_shadow mr7"
        >
          
          
        
        </button>
         
        <button
          class="button-clasic button_theme-blue button_theme-blue_shadow"
        >
          
          
        
        </button>
      </div>
    </div>
  </div>
</div>
`;
